playbooks/server-install.yml

```diff
@@ -2,6 +2,10 @@
 - name: Setup Libvirt on the target node
   hosts: all
   become: yes
+  vars:
+    snapshot_timestamp: "20250701T120000Z"
+    docker_version_string: "5:28.3.0-1~debian.12~bookworm"
```

Comment:
Can you add inline comments on how to grab the latest snapshot version and the latest docker versions?